### DISCUSSION TITLE
Ability to ignore links that have occurred once in a note already

### DISCUSSION THREAD

**Discussion by sscrewston** [TfTHacker/obsidian42-strange-new-worlds#59] [Ideas]
I tend to link the same note multiple times in another note because I like linking everything but I was wondering if there's a way for strange new worlds to just ignore other references in that note if I've already referenced it once. When I hover over the reference counter in strange new world and scroll through link references I don't want to keep scrolling through references made in the same note over and over again if that makes sense.

**Comment by TfTHacker** [comment]
I think this is what you are looking for:

https://github.com/TfTHacker/obsidian42-strange-new-worlds#snw-frontmatter-directives

**Reply by sscrewston** [reply]
Unfortunately not, I want my notes to be included in SNW index and I want it enabled on all pages. I just don't want every single (of the same) link in one page to be indexed, only one at least. I guess one link reference per note or the first one? I'm not sure if that's practical or not.

**Comment by pictoys** [comment]
i was about to ask for the same. if a block has multiple mentions of the same it shows up twice. would be great to fix this.
